test/cli/listSecrets.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import json
+import argparse
+import base64
+import time
+import random
+import hmac
+from hashlib import sha256
+from collections import OrderedDict
+import urllib.parse
+
+import _utils_
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--url', type=str, help='the address of the ehsm_kms_server', required=True)
+    parser.add_argument('--secretName', type=str, help='the name of the secret')
+    args = parser.parse_args()
+
+    base_url = args.url + "/ehsm?Action="
+    print(base_url)
+    return base_url, args.secretName
+
+def listSecrets(base_url, secretName):
+    payload = OrderedDict()
+    if secretName != None:
+        payload["secretName"] = secretName
+    params = _utils_.init_params(payload)
```

Comment:
if secretName != None:
        payload["secretName"] = secretName
        params = _utils_.init_params(payload)
else:
        params = _utils_.init_params()

Reply:
will be change